README.md

```diff
@@ -31,15 +31,15 @@ The `Dockerfile` present in this repository and its respective `twilio/twilio-no
 
 ## Getting help
 
-If you need help installing or using the library, please contact Twilio Support at help@twilio.com first. Twilio's Support staff are well-versed in all of the Twilio Helper Libraries, and usually reply within 24 hours.
+If you need help installing or using the library, please check the [Twilio Support Help Center](https://support.twilio.com) first.
```

Comment:
If you need help installing or using the library, please check the [Twilio Support Help Center](https://support.twilio.com) first, and file a support ticket there if you don't find an answer to your question.